File: Em Python/1043.py
a, b, c = map(float, input().split())

if b - c < a and a < b + c and a - c < b and b < a + c and a - b < c and c < a + b :
    print("Perimetro = %0.1f" % (a+b+c))
else:
    print("Area = %0.1f" % (((a+b)*c)/2))

File: Em C/1043.c
#include <stdio.h>

int main(int argc, char** argv)
{
	float a,b,c;
	scanf("%f %f %f",&a,&b,&c);
	if(b - c < a && a < b + c && a - c < b && b < a + c && a - b < c && c < a + b){
		printf("Perimetro = %.1f\n",a+b+c);
	}else{
		printf("Area = %.1f\n",((a+b)/2)*c);
	}
	return 0;
}